app/views/content/support-for-early-career-teachers.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Support for early career teachers (ECTs)"
+heading: "Support for early career teachers (ECTs)"
+description: |-
+  Find out about the support early career teachers get when they start teaching, including mentor support and training based on the early career framework (ECF).
+related_content:
+    Initial teacher training : "/train-to-be-a-teacher/initial-teacher-training"
+    Advice from an early career teacher : "/blog/being-myself-in-the-classroom"
+    Managing behaviour in the classroom : "/blog/managing-behaviour-in-the-classroom"
+    Becoming a numeracy coordinator in my first year of teaching : "/blog/abigails-career-progression-story"
+promo_content:
+    - content/train-to-be-a-teacher/promos/get-adviser-promo
+---
+
+You’ll be known as an early career teacher (ECT) for your first 2 years in teaching. The term early career teacher (ECT) replaces newly qualified teacher (NQT).
+
+As part of these 2 years, you’ll have an induction period underpinned by the [early career framework (ECF)](https://www.gov.uk/guidance/how-the-early-career-framework-ecf-supports-induction) which explains the training and support ECTs are entitled to.
+
+You’ll receive mentoring and training to help you develop your teaching practice. This will allow you to build the skills and knowledge you need to feel confident and empowered in your classroom.
+
+This won’t impact your pay or progression opportunities. You’ll still be able to progress on the pay scale during your induction period.
+
+## Early career framework-based training
+
+You’ll receive training based on the early career framework (ECF) as part of your induction period. This is called ECF-based training.
+
+ECF-based training provides a bridge between your initial teacher training (ITT) and your teaching career. Your ITT helps you qualify as a teacher, and the ECF-based training helps you further develop your teaching practice and working habits.
+
+As part of this training, you’ll have a dedicated mentor, online or face to face training, and paid time away from classroom teaching to focus on your development.
+
+### Your mentor
+
+You’ll be given a mentor during your induction period.
+
+Your mentor will:
+
+* provide one to one support and feedback
+* work with your school to make sure you’re receiving a high quality induction programme
+* provide or arrange mentoring and coaching around specific subject areas
+* take prompt, appropriate action if you’re having difficulties
```

Comment:
I'd question whether we need to say 'appropriate' - surely they wouldn't take inappropriate action!

(But I don't feel strongly about this)

Reply:
Let's leave as is then!